vcpkg/users/examples/versioning.getting-started.md

```diff
@@ -157,7 +157,7 @@ If you want to upgrade your dependencies, you can bump the minimum version const
 { "builtin-baseline": "3426db05b996481ca31e95fff3734cf23e0f51bc" }
 ```
 
-This field declares the versioning baseline for all ports. Setting a baseline is required to enable versioning, otherwise you will get the current versions on the ports directory. You can run 'git rev-parse HEAD' to get the current commit of vcpkg and set it as the builtin-baseline. See the [`builtin-baseline` documentation](../manifests.md#builtin-baseline) for more information.
+This field declares the versioning baseline for all ports. Setting a baseline is required to enable versioning, otherwise you will get the current versions on the ports directory. You can run 'git rev-parse HEAD' to get the current commit of vcpkg and set it as the builtin-baseline. See the [`"builtin-baseline"` documentation](../vcpkg-json.md#builtin-baseline) for more information.

```

Comment:
Minor correction: "otherwise you will get the current versions **in** the ports directory". 

Might be worth italicizing "ports" in "ports directory" as well since it's the actual name of that directory.